docs/api-auth.md

```diff
@@ -7,13 +7,15 @@ There are two ways to authenticate against the DCE APIs:
 
 ## Using Cognito
 
-Cognito is used to allow an admin to add users to DCE.  This can be done by using the Cognito User Pool.  Additionally any IdP supported by Cognito User Pools can also be used.
+Cognito is used to allow an admin to add users to DCE.  This can be done by using the Cognito User Pool.  This section will walk through setting this 
```

Comment:
> Cognito is used to allow an admin to add users to DCE.

That not quite right. Cognito is used to authenticate and authorize DCE API users. 
Maybe the terminology is just getting confusing here, as we're talking about _admins_ in a couple different contexts (ie, _System admin_ can use Cognito to manage _API users_. But there's no concept of _API Admins_ managing _DCE users_)

Reply:
Good catch, I'll rewrite this section.